app/src/main/java/a3/com/convo/activities/PlayGameActivity.java

```diff
@@ -16,10 +17,10 @@
     private FriendsFragment friendsFrag;
```

Comment:
Why are we saving all these fragments in memory? Its a BIG NO NO to save fragments/activities in memory. Lets refactor this to not define fragment as member variable